### DISCUSSION TITLE
adding openvdb under third_party

### DISCUSSION THREAD

**Discussion by unicomp21** [google/filament#6886] [General]
objections?

**Comment by unicomp21** [comment]
and if not openvdb, what about nanovdb?

**Comment by romainguy** [comment]
We would only add a third part dependency if it’s used by Filament or its tools. Why do you need a vdb reader in Filament’s source tree?

**Reply by unicomp21** [reply]
generating procgen assets on the fly within filament.  it's not about reading asset files, it's about generating them on the fly.

**Comment by unicomp21** [comment]
due to bandwidth constraints, when doing metaverse type stuff, procgen starts making a lot of sense?
